src/platform/datapath_winuser.c

```diff
@@ -731,6 +731,16 @@ CxPlatDataPathQuerySockoptSupport(
     return Status;
 }
 
+//
+// CloseHandle has an incorrect SAL annotation, so call through a wrapper.
+//
+_IRQL_requires_max_(PASSIVE_LEVEL)
+inline
+void
+CxPlatCloseHandle(_Pre_notnull_ HANDLE Handle) {
+    CloseHandle(Handle);
```

Comment:
Don't you need to suppress the SAL error? Can you also put this into the platform header instead?

Reply:
Nope, no need to suppress the SAL error, as just calling it through a function is enough to work around the bug. I can move it to the header though.

Reply:
Basically, the bad SAL annotation was thrashing ALL other handle values in the loop. Through a wrapper, it doesn't know its in a loop, so it doesn't thrash.